src/components/Card/Card.tsx

```diff
@@ -77,7 +97,10 @@ export const Card = React.forwardRef<HTMLDivElement, CardProps>(function Card(pr
                 },
                 className,
             )}
-            onClick={isClickable ? onClick : undefined}
+            onClick={handleClick}
+            onKeyDown={handleKeyDown}
```

Comment:
Let's use button tag for interactive card to get rid of this handler and eslint errors. And add `role="button"`.